docs/advanced-features/custom-server.md

```diff
@@ -39,9 +39,9 @@ app.prepare().then(() => {
     const { pathname, query } = parsedUrl
 
     if (pathname === '/a') {
-      app.render(req, res, '/b', query)
-    } else if (pathname === '/b') {
       app.render(req, res, '/a', query)
+    } else if (pathname === '/b') {
```

Comment:
I thought that was the whole point of the example: to render another page that did not match the path?

Reply:
The example explains that in the README but in this doc page it looks like a typo, hence my suggestion above